### ISSUE TITLE
poll location for screenreaders

### ISSUE THREAD

**Issue by mtsonline** [bigbluebutton/bigbluebutton#20621]
**Describe the bug**

when using a screenreader during a BBB session polls are shown above the actionbar.
to read and answer the poll one has to find it first, by moving the cursor to the bottom and then go up step by step until the poll can be found.

It would be much easier if the poll was placed at the end of the dom tree so it is read / shown to screenreaaders at the bottom of the page.

It also should have a header - as e.g. the members title has (if not already)

**To Reproduce**
use a screenreader during a session and try to find the opened poll question and answer buttons :-)

**Expected behavior**

Easiest would be, if a area toast is used, so the question is read out automatically. 
Additional a header at the top of the poll would be helpful to jump to the Poll area.
And last, if it is placed at the end of the dom tree, screenreaders would be able to find it at the end - so one can jump there easily, even if the screenreader did not read out automatically, and one does not know, that a heading is given to find it quickly.

**Comment by antobinary** [comment]
Hi @mtsonline,
Thank you for sharing this feedback in the community call earlier today and for opening an issue about it!

**Comment by mtsonline** [comment]
I think this is still an issue. Maybe a coursor trap would be a good solution or at least a focus() on the beginning of the poll.
